app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/login/WelcomeTitle.kt

```diff
@@ -2,39 +2,88 @@ package ch.epfl.cs311.wanderwave.ui.components.login
+      modifier = modifier.fillMaxWidth()) {
+        AnimatedIcon()
         Text(
             text = stringResource(id = R.string.welcome_title),
             style = MaterialTheme.typography.displayLarge,
             textAlign = TextAlign.Center,
             color = MaterialTheme.colorScheme.onSurface,
             modifier = Modifier.testTag("welcomeTitle"))
-        Spacer(modifier = Modifier.height(10.dp))
         Text(
             text = stringResource(id = R.string.welcome_subtitle),
             style = MaterialTheme.typography.bodyLarge,
             textAlign = TextAlign.Center,
             color = MaterialTheme.colorScheme.onSurface,
             modifier = Modifier.testTag("welcomeSubtitle"))
+        Spacer(modifier = Modifier.height(10.dp))
+        Row(
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = Alignment.CenterVertically) {
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Red)
+              LoginScreenPulse(startColor = Color.Red, endColor = Color.Magenta, reverse = false)
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Red,
+                  modifier = Modifier.size(20.dp))
+            }
+        Row(
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = Alignment.CenterVertically) {
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Green)
+              LoginScreenPulse(startColor = Color.Green, endColor = Color.Magenta, reverse = false)
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Green,
+                  modifier = Modifier.size(20.dp))
+            }
+        Row(
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = Alignment.CenterVertically) {
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Blue)
+              LoginScreenPulse(startColor = Color.Blue, endColor = Color.Magenta, reverse = false)
+              Icon(
+                  painter = painterResource(id = R.drawable.hexagon),
+                  contentDescription = stringResource(id = R.string.circle_icon),
+                  tint = Color.Blue,
+                  modifier = Modifier.size(20.dp))
+            }
```

Comment:
While these animated lines are fancy, I don't really see the point of having them on the login screen. From a UX perspective, the user should be directed to press the login button and not really do much else on the login screen.

Reply:
Agreed.